Script: GitHub how to (#9)
pupilfirst/wd201-scripts: introduction-to-nodejs/github-how-to/README.md

# Script

> In this video, I'm going to show you, how to use the `git` command line interface in any application.

Reviewer: ```suggestion
In this video, I'm going to show you, how to use the `git` command line interface.
```

> Before continuing, let me quickly mention why this course requires you to use git. I've already told you that, git is the defacto standard for version control in the coding industry. Regardless of which language you use in your work, you'll almost certainly be using git as your version control system. So it's pretty much necessary for every developers out there.

Reviewer: ```suggestion
Before continuing, let me quickly mention why this course requires you to use git. Git is the de facto standard for version control in the software industry. Regardless of language you use in your work, you'll almost certainly be using git as your version control system. So it's pretty much necessary for every developer out there.
```
